leaf-ui/rappid-extensions/ConfigInspector.js

```diff
@@ -124,11 +124,14 @@ var Config = Backbone.View.extend({
 
     events: {
         'click .config-elements': 'switchConfig',
-        'click .deleteconfig-button': 'removeConfig',
         'click .dropdown-button' : 'toggleDropdown',
         'dblclick .config-elements': 'rename',
         'blur .config-input': 'setConfigName',
-        'keyup .config-input': 'checkForEnter'
+        'keyup .config-input': 'checkForEnter',
+        'click .deleteconfig-button': function(e) {
+            this.switchConfig(e);
```

Comment:
We don't want to call switchConfig here though, because that will switch the deleted config to being selected

Reply:
I called it there because without the switchConfig in line 132, there were multiple configs highlighted as the previously highlighted config was not being unhighlighted. Do you think I should try to use another switchConfig or something similar elsewhere or remove the original switchConfig and figure something else out?

Reply:
You can remove switchConfig(e) from here now since you check to see if the model is selected before switching to another model in the remove function